src/features/fee-row/fee-row.tsx

```diff
@@ -102,57 +103,69 @@ export function FeeRow(props: FeeRowProps): JSX.Element {
-                color={color('text-caption')}
-                onClick={() => openInNewTab(URL)}
-                size="14px"
-              />
-            </Stack>
-          </Tooltip>
-        </Stack>
-        {isCustom ? (
-          <CustomFeeField setFieldWarning={setFieldWarning} />
-        ) : !fee ? (
-          <LoadingRectangle width="50px" height="10px" />
-        ) : (
-          <Suspense fallback={<>0.00</>}>
+    <>
+      <Stack spacing="base">
+        <SpaceBetween position="relative">
+          <Stack alignItems="center" isInline>
             <Caption>
-              <TransactionFee />
+              <Flex>Fees</Flex>
             </Caption>
-          </Suspense>
+            <Stack _hover={{ cursor: 'pointer' }}>
+              <FeeEstimateItem
+                hasFeeEstimations={!feeEstimationsQueryError}
+                index={selected}
+                onClick={!feeEstimationsQueryError ? () => setIsOpen(true) : undefined}
+              />
+              <FeeEstimateSelect
+                items={feeEstimations}
+                onClick={handleSelectedItem}
+                setIsOpen={setIsOpen}
+                visible={isOpen}
+              />
+            </Stack>
+            <Tooltip label={FEES_INFO} placement="bottom">
+              <Stack>
+                <Box
+                  _hover={{ cursor: 'pointer' }}
+                  as={FiInfo}
+                  color={color('text-caption')}
+                  onClick={() => openInNewTab(URL)}
+                  size="14px"
+                />
+              </Stack>
+            </Tooltip>
+          </Stack>
+          {isCustom ? (
+            <CustomFeeField setFieldWarning={setFieldWarning} />
+          ) : !fee ? (
+            <LoadingRectangle width="50px" height="10px" />
+          ) : (
+            <Suspense
+              fallback={
+                <>
+                  {stacksValue({
+                    value: 0,
+                    fixedDecimals: true,
+                  })}
+                </>
+              }
+            >
+              <Caption>
+                <TransactionFee />
+              </Caption>
+            </Suspense>
+          )}
+        </SpaceBetween>
+        {errors.txFee && (
+          <ErrorLabel data-testid={SendFormSelectors.InputCustomFeeFieldErrorLabel}>
+            <Text lineHeight="18px" textStyle="caption">
+              {fieldError}
+            </Text>
+          </ErrorLabel>
         )}
-      </SpaceBetween>
-      {errors.txFee && (
-        <ErrorLabel data-testid={SendFormSelectors.InputCustomFeeFieldErrorLabel}>
-          <Text lineHeight="18px" textStyle="caption">
-            {fieldError}
-          </Text>
-        </ErrorLabel>
-      )}
-      {!errors.txFee && fieldWarning && <WarningLabel>{fieldWarning}</WarningLabel>}
-    </Stack>
+        {!errors.txFee && fieldWarning && <WarningLabel>{fieldWarning}</WarningLabel>}
+      </Stack>
+      <HighFeeDrawer />
```

Comment:
I don't think we should place the `<HighFeeDrawer />` in the FeeRow component. The fee row is to display the UI to select a fee.

Displaying the drawer when the fee is too high triggers global form behaviour, so I think it'd be better placed in the top component of the form.

Reply:
Yep, I can move it, wasn't sure on this bc now I have to put it in 2 places bc there are 2 forms that use the fee row.